### PR TITLE
interpret() as a method

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
Make interpret as a method so we can use do more stuff with it

### CODE REVIEW

File: R/interpret.R [hunk 1]
```diff
@@ -104,7 +104,12 @@ is.rules <- function(x) inherits(x, "rules")
 #' interpret(c(0.35, 0.15), c("small" = 0.2, "large" = 0.4), name = "Cohen's Rules")
 #' interpret(c(0.35, 0.15), rules(c(0.2, 0.4), c("small", "medium", "large")))
 #' @export
-interpret <- function(x, rules, name = attr(rules, "rule_name")) {
+interpret <- function(x, ...){
+  UseMethod("interpret")
+}
+
+
+interpret.numeric <- function(x, rules, name = attr(rules, "rule_name")) {
```
Comment (strengejacke) [inline R/interpret.R:112]: you need to add `...` as arg, too.

**Review by strengejacke — APPROVED** [review]

**Comment by mattansb** [comment]
Should the various methods be implemented in their respective packages? (Or perhaps it is finally time to move `interpret()` to report altogether...?)

**Comment by DominiqueMakowski** [comment]
😠 I moved this chunk specifically *from* report to here 😅  effect size interpretation is conceptually one of the purpose, and one of usage of effect sizes (and even though rules-of-thumb-based labelling surely has major limitations, its super wide-spread adoption suggests its usefulness - or lack of convenient alternative - nonetheless). And to be honest, for the plethora of SEM-related indices of fit, is there more than 0 people in the world who can make sense out of them without such rules of thumb? "Ah yes my BLABLOBLI is 0.67893, I totally get what it means"

It's a purely objective and explicit process of attribution of a label to a value based on some rules that are published already, so it  has its place here rather than in report (which uses it to create a more "subjective" description)

So it'd be nice in the future if interpret() can be used for model_parameters() and model_performance() objects down the line to automatically run the correct interpret function on the values. It's a great feature, trust me ^^

**Comment by mattansb** [comment]
Not arguing against any of that (: 
But since `interpret()` is is used for things other than effect sizes (e.g., all of the SEM BLABLOBLI), and since it is used extensively by `report`,  ...

**Comment by DominiqueMakowski** [comment]
In my mind measures of fit are kind of practically close to effect sizes, although not effect sizes per se but their inform us on the "performance" of the model/parameters and can be used to make interpretations, so makes sense to keep the functions here 😁 

can merge this?

**Comment by mattansb** [comment]
Dom says: so makes sense to keep the functions here 😁
Mattan hears: I'd rather you maintain this 🤪

Alright, I'm too sleep deprived to argue this point further - will resume in the future 😉

**Comment by DominiqueMakowski** [comment]
haha nah don't worry I'll help if need be 😁 (but I dont see much maintaining happening)

**Pull request merged**